client/src/controllers/CountController.ts

```diff
@@ -60,7 +60,7 @@ export class CountController extends Controller<HTMLFormElement> {
   declare readonly totalTarget: HTMLElement;
 
   connect() {
-    this.count();
+    document.addEventListener('change', () => this.count());
```

Comment:
This was added to allow **w-count** to select dynamic elements, so please ensure it doesn't cause any issues.

Reply:
We should just add this event listener at the HTML. Via a Stimulus action.